src/oic/utils/sdb.py

```diff
@@ -1091,3 +964,117 @@ def read(self, token):
             raise WrongTokenType("Not a grant token")
 
         return self._db[key]
+
+    def get_by_sub(self, sub):
+        return self._db.get_by_sub(sub)
+
+    def make_smid(self, sid):
+        """
+        Create a session management ID.
+
+        :param sid:
+        :return: A session management ID
+        """
+        return hashlib.sha256(
+            "{}{}".format(sid, self.sm_salt).encode("utf-8")
+        ).hexdigest()
+
+    def get(self, attr, val):
+        return self._db.get(attr, val)
+
+    def get_by_uid(self, uid: str) -> List[str]:
+        return self._db.get_by_uid(uid)
+
+    def get_uid_by_sub(self, sub):
+        return self._db.get_uid_by_sub(sub)
+
+    def get_uid_by_sid(self, sub):
+        return self._db.get_uid_by_sid(sub)
+
+
+def session_get(db, attr, val):
```

Comment:
It looks like these are not used that much. There is probably quite a lot of calls to the old API?

Reply:
Might become more used when we deal with the provider side.

Reply:
Could be, we will see. Maybe we will need to do a followup cleanup to properly use them everywhere.